src/main/java/com/aventstack/extentreports/Report.java

```diff
@@ -100,8 +100,6 @@ synchronized void addNode(Test node) {
     }
     
     synchronized void addLog(Test test, Log log) {
-        collectRunInfo();
```

Comment:
If we remove this line of code, consider for very long tests - the user will not have up-to-date information after each log.